src/main.c

```diff
@@ -76,11 +78,17 @@ void app_main(void) {
     uint16_t sw = ERR_NOT_IMPLEMENTED;
     bool isNewCall = false;
 
-    ui_idle();
+#ifdef HAVE_SWAP
+    if (!G_called_from_swap)
+#endif
+    {
+        ui_idle();
```

Comment:
why do we have both in nbgl ?

Reply:
Those are not the same. These functions were already called, I have not change that, just adapt the calls location for Standard App